Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM debian:stable
+
+RUN apt update -y
+
+RUN apt install -y alien
+
+WORKDIR /tmp
+
+RUN mkdir img
+
+RUN mkdir src
+
+COPY img/ img
+
+COPY src/ src
+
+COPY zerotier-gui.desktop .
+
+COPY make_deb.sh .
+
+COPY README.md .
+
+COPY LICENSE .
+
+RUN chmod +x make_deb.sh
+
+RUN ./make_deb.sh
+
+RUN alien -r ZeroTier-GUI.deb
+
+RUN mv zerotier-gui-1.3.0.r.-2.noarch.rpm ZeroTier-GUI.rpm
+
+RUN ls -la
```

Comment:
I don't understand why this `ls` is needed.

Reply:
> I don't understand why this `ls` is needed.

There is no need in fact (I forgot the instruction haha)